generation_config.yaml

```diff
@@ -167,6 +167,14 @@ libraries:
   GAPICs:
   - proto_path: google/appengine/v1
 
+- api_shortname: apphub
```

Comment:
Which script do you use to populate this yaml?

Reply:
I manually added these libraries, the parameters are copied from new library generation PR.